htdocs/comm/propal/index.php

```diff
@@ -119,7 +119,7 @@
 				$companystatic->name = $obj->name;
 				$companystatic->client = $obj->client;
 				$companystatic->code_client = $obj->code_client;
-				$companystatic->code_fournisseur = $obj->code_fournisseur;
+				$companystatic->code_fournisseur = isset($obj->code_fournisseur)?$obj->code_fournisseur:'';
```

Comment:
@bb2a perhaps you can add code_fournisseur in the sql request to fix

Reply:
Yes and fix in v16